tests/test_jointcal_hsc.py

```diff
@@ -150,6 +152,31 @@ def test_jointcalTask_2_visits_simpleMagnitude(self):
 
         self._testJointcalTask(2, None, None, pa1, metrics=metrics)
 
+    def test_jointcalTask_2_visits_simpleMagnitude_colorterms(self):
+        """Test that colorterms are applied and change the fit."""
+        pa1, metrics = self.setup_jointcalTask_2_visits_simplePhotometry()
+        self.config.photometryModel = "simpleMagnitude"
+        self.config.applyColorTerms = True
+        # The values in this library were taken from obs_subaru as of stack version 16.0
+        self.config.colorterms = ColortermLibrary(data={
+            "sdss*": ColortermDict(data={
+                'r': Colorterm(primary="r", secondary="i", c0=0.00231810, c1=0.01284177, c2=-0.03068248)
+            })
+        })
+        # Final chi2 should be different, but I don't have an a-priori reason
+        # to expect it to be larger or smaller.
```

Comment:
If the color terms are doing what they should be doing, and if the reference stars have a wide enough color range (which may not be the case!) then I would expect the chi2 to be slightly smaller.  But if the color corrections are subtle and it's not good statistics, then I can see how it could increase.

Reply:
In this case, I think we might be in the "not good statistics" situation: there are only two hsc visits going into this, and only a handful of ccds, and it's a "simple" model, so this is mostly just to demonstrate that it "does something".